fdbclient/vexillographer/fdb.options

```diff
@@ -292,6 +292,11 @@ description is not currently required but encouraged.
                 description="Asks storage servers for how many bytes a clear key range contains. Otherwise uses the location cache to roughly estimate this." />
     <Option name="bypass_unreadable" code="1100"
                 description="Allows ``get`` operations to read from sections of keyspace that have become unreadable because of versionstamp operations. These reads will view versionstamp operations as if they were set operations that did not fill in the versionstamp." />            
+    <Option name="use_grv_cache" code="1101"
+            description="Allows this transaction to use cached GRV from the database context. Defaults to off. Upon first usage, starts a background updater to periodically update the cache to avoid stale read versions." />
+    <Option name="skip_grv_cache" code="1102"
+            description="Specifically instruct this transaction to NOT use cached GRV. Primarily used for the read version cache's background updater to avoid attempting to read a cached entry in specific situations."
+            hidden="true"/>
```

Comment:
Given that the GRV cache is off for each transaction by default, is `skip_grv_cache` actually necessary? The most compelling case I could come up for it would be if there were a database option that could set up transactions to use the GRV cache by default, and then the `skip_grv_cache` flag would be necessary to turn it off.

Reply:
Yes, that's exactly the case this is being used for. There is a debug knob that can make all transactions use the GRV cache, and this is to let the background updater bypass it

Reply:
Note also that this usage is internal, and so the option is meant for internal use and marked hidden.